ZitiUpdateService/UpdateService.cs

```diff
@@ -887,6 +899,10 @@ public partial class UpdateService : ServiceBase {
 				dnsProbeTimer.Start();
 				Logger.Info("DNS Probe enabled");
 			}
+			if (_installationReminder != null) {

```

Comment:
This code block is added to send notification to the UI, when user restarts ziti-tunnel